food_vendor_app/models.py

```diff
@@ -46,7 +62,7 @@ class Menu(models.Model):
         ('DAIRY-FREE', 'Dairy-Free')
     ]
 
-    vendor_id = models.ForeignKey(UserProfile, on_delete=models.CASCADE)
+    vendor = models.ForeignKey(VendorProfile, on_delete=models.CASCADE)
```

Comment:
I can see you have a separate model for vendors. No model for Customers? Can see Customers are referencing User model

Reply:
the UserProfile has business name? meaning a customer will have a business name?